integrations/jenkins/.port/resources/port-app-config.yaml

```diff
@@ -0,0 +1,35 @@
+createMissingRelatedEntities: true
+deleteDependentEntities: true
+resources:
+  - kind: jenkinsJob
+    selector:
+      query:  "true"
+    port:
+      entity:
+        mappings:
+          identifier: 'if .url | contains("://") then (.url | split("://")[1] | sub("^.*?/"; "")) else .url end | sub("%20"; "-"; "g") | sub("/"; "-"; "g") | .[:-1]'
+          title: .displayName
+          blueprint: '"jenkinsJob"'
+          properties:
+            jobName: .fullName
+            url: '.url | if test("://") then sub("^[^/]+//[^/]+"; "") else . end'
```

Comment:
what is the purpose of this jq?

Reply:
Oh, this is a left-over from when I was trying to handle the two json schemas of `events` and `api`. I will simplify it.